includes/Core/Assets/Assets.php

```diff
@@ -383,6 +384,12 @@ private function get_assets() {
 					},
 				)
 			),
+			new Google_Charts(
+				'googlesitekit-google-charts',
+				array(
+					'in_footer' => false,
+				)
+			),
```

Comment:
I don't think it's worth introducing a class for this. We can define the details and pass them to a `Script` just like we do elsewhere. The `Script` class should also work for external assets.
```suggestion
			new Script(
				'googlesitekit-google-charts',
				array(
					'src'          => 'https://www.gstatic.com/charts/loader.js',
					'in_footer'    => false,
					'before_print' => function() {
						wp_add_inline_script(
							'googlesitekit-google-charts',
							'google.charts.load( "current", { packages: [ "corechart" ] } );'
						);
					},
				)
			),
```

Reply:
Oh.. Nice! Updated :smile: